docs/content/releases.md

```diff
@@ -7,6 +7,36 @@ toc: true
 docs: "DOCS-616"
 ---
 
+## NGINX Ingress Controller 2.2.0
+
+12 Apr 2022
+
```

Comment:
We'll need an OVERVIEW section here also with details of cert-manager integration, IPv6 support, and deprecation of the preview policy flag. I'll try and cobble something together today 😄

Reply:
Something like (but I'll defer to the experts to flesh out the details and refine the wording):
```suggestion

OVERVIEW:

* Support for automatic provisioning and management of Certificate resources for VirtualServer resources using [cert-manager](https://cert-manager.io/docs/). Examples for configuring cert-manager with NGINX Ingress Controller can be found [here](https://github.com/nginxinc/kubernetes-ingress/tree/v2.2.0/examples/custom-resources/certmanager). Please note that ACME type Issuers are not yet supported for use with VirtualServer resources.

* Full support for IPv6 using the NGINX Ingress Controller Ingress and [VirtualServer and VirtualServerRoute](https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources) resources. 

* The [-enable-preview-policies](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#-enable-preview-policies) cli argument has been deprecated and is no longer required for the usage of any Policy resource type. This argument will be removed completely in v2.4.0.

* A new [-enable-oidc](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#-enable-oidc) cli argument has been added to enable OIDC policies. Previously, this behaviour was achieved through the usage of the `-enable-preview-policies` cli argument. 

```

Reply:
-enable-preview-policies will be removed in 2.6 - it is a 3 release deprecation notice prior to removal.

Reply:
For IPv6 - list the VS/VSR resources prior to the Ingress resource in the explanation.  (we should begin to always lead with our CRDs)